__tests__/index.test.js

```diff
@@ -128,6 +128,26 @@ describe("/api/articles", () => {
             expect(response.body.message).toBe("bad request");
           });
       });
+      test("returns articles that match the topic query", () => {
+        return request(app)
+          .get("/api/articles?topic=cats")
+          .expect(200)
+          .then((response) => {
+            const articles = response.body.articles;
+            expect(articles.length).toBeGreaterThan(0);
+            articles.forEach((article) => {
+                expect(article.topic).toEqual("cats");
+            })
+          });
+      });
```

Comment:
We should write a test to check when the topic is valid and exists but has not articles, like for paper :)